src/v2/Apps/Artwork/Components/OtherWorks/index.tsx

```diff
@@ -86,6 +96,13 @@ export const OtherWorks = track()(
               )
               return (
                 <Box key={`Grid-${index}`} data-test={contextModule}>
+                  {artistSeriesIsEnabled &&
```

Comment:
We want to attach this rail so it comes before the 'Other works from this artist' grid (and if the lab feature is enabled).